### PR TITLE
Skip btn comment onboarding

### LINKED ISSUE #6366: [Feature] Skip button comments onboarding

### PR DESCRIPTION
**Pull request by rishabhsaxena**
Fixes #6366

## Test coverage results :test_tube:
<details><summary>:green_circle: Total coverage has increased</summary>

    // Code coverage diff between base branch:release and head branch: skip-btn-comments-onboarding 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | total | 51.7 **(0)** | 33.64 **(0.01)** | 29.53 **(0)** | 52.35 **(0)**
 :green_circle: | app/client/src/constants/messages.ts | 69.11 **(-0.01)** | 100 **(0)** | 6.69 **(-0.03)** | 76.06 **(0.07)**
 :green_circle: | app/client/src/utils/autocomplete/TernServer.ts | 50.71 **(0.24)** | 38.6 **(0.88)** | 36.21 **(0)** | 55.35 **(0.27)**</details>

### CODE REVIEW

File: app/client/src/components/ads/ShowcaseCarousel.tsx [hunk 1]
```diff
@@ -2,9 +2,10 @@ import React, { useState } from "react";
 import Button, { Category, Size } from "components/ads/Button";
 
 import styled from "styled-components";
-import { createMessage, NEXT, BACK } from "constants/messages";
+import { createMessage, NEXT, BACK, SKIP } from "constants/messages";
 import { useTransition, animated } from "react-spring";
 import { useCallback } from "react";
+import { useEffect } from "react";
```
Comment (pranavkanade) [inline app/client/src/components/ads/ShowcaseCarousel.tsx:8]: nit pic: import at the top.

File: app/client/src/comments/CommentsShowcaseCarousel/index.tsx [hunk 2]
```diff
@@ -165,6 +170,8 @@ export default function CommentsShowcaseCarousel() {
   const profileFormValues = useSelector(getFormValues(PROFILE_FORM));
   const profileFormErrors = useSelector(getFormSyncErrors("PROFILE_FORM"));
   const isSubmitDisabled = Object.keys(profileFormErrors).length !== 0;
+  const [activeIndex, setActiveIndex] = useState(0);
```
Comment (pranavkanade) [inline app/client/src/comments/CommentsShowcaseCarousel/index.tsx:173]: I suppose we should create an enum to keep track of the different steps in onboarding, it'll improve readability.

**Review by pranavkanade — APPROVED** [review]
couple of suggestions, rest LGTM!

**Review by pranavkanade — APPROVED** [review]

**Comment by rishabhsaxena** [comment]
Merging this since the failing test case, `API_Edit_spec` isn't related to this PR and we need to merge this on a priority

**Pull request merged**